### DISCUSSION TITLE
发一张芙芙美照

### DISCUSSION THREAD

**Discussion by lsziumlan** [share121/inter-knot#5416] [原神]
![1723719456546](https://github.com/user-attachments/assets/2fdea826-a06b-4860-9897-d95e1e04226b)

**Comment by aomengxin** [comment]
赞，拿了

**Comment by sanliangdi** [comment]
芙芙可爱

**Comment by Jiturine** [comment]
有无原图画师啊，求

**Comment by hutao48** [comment]
![file-DeZ2SzdoahJ6PtTyS0jc3atO](https://github.com/user-attachments/assets/a1cfb32b-0eed-4a4d-a5a9-de02100f8f3a)
技术不太行
~~我是废物~~
